### PR TITLE
Add Tutorial: Setup Squid with cache, antivirus and SSL-MITM mode on Managed Server

### PR DESCRIPTION
**Pull request by alexanderknerlein**

### CODE REVIEW

File: tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+
+### Step 4.3 - Test it
+
++ Please test some SSL failures at [badssl.com](https://badssl.com/)
++ Please visit [eicar.org](https://www.eicar.org/) to test the malware detection
+
+
+## Conclusion
+
+Now you got a basic introduction how to Install and configure a Squid proxy with a bunch of additional services on a Managed Server. It comes to your attention to secure the proxy, stay up to date and tweak the configuration.
```
Comment (BackInBash) [inline tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md:292]: Your conclusion contains unanswered questions maybe you should answer them and expand the tutorial.

File: tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+And finally download and compile the SquidClamav sources.
+
+```bash
+wget https://github.com/darold/squidclamav/archive/v7.1.tar.gz
+tar xzvf v7.1.tar.gz
+cd squidclamav-7.1
+./configure --prefix=/usr/home/holu/squid --with-c-icap=/usr/home/holu/squid
+make -j $(grep "^cpu\ cores" /proc/cpuinfo | uniq | sed s/[^0-9]//g)
+make install
+```
```
[the remark this answers is not in the document]
Comment (alexanderknerlein) [reply tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md:74]: This tutorial is especially made for Managed Servers. We can't get root access on a Managed Server (we can't use apt). The only way to install it is the source installation in the userspace. Some packages are possible [custom software installer](https://wiki.hetzner.de/index.php/KonsoleH:Installation_g%C3%A4ngiger_Software_(Managed_Server)/en), but not this one.

File: tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md (hunk 1 above)
Comment (alexanderknerlein) [reply tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md:292]: Maybe, but I just wanted to describe a really basic configuration here. Only the necessary things to get it to work. This tutorial shouldn't do everything for you.

File: tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (n-se) [reply tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md:74]: You could mention where to find newer versions of this software ( https://github.com/darold/squidclamav/releases )

File: tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md (hunk 1 above)
Comment (n-se) [reply tutorials/setup-squid-with-cache-antivirus-and-ssl-mitm-mode-on-managed-server/01.en.md:292]: Can you add some links to the documentation page(s)?

**Review by n-se — APPROVED** [review]
LGTM

**Comment by fdrechsler** [comment]
Found some Issues, please don't release. 
Exact Discussions following. Give me some days till i find some time to write them down.

**Comment by rook59** [comment]
@fdrechsler any updates on what needs to be changed here?

**Comment by rook59** [comment]
As discussed nothing needs to be changed. @alexanderknerlein is going to update the tutorial once again due to version changes.

**Comment by alexanderknerlein** [comment]
The src versions are now up to date. I'm done.

**Review by rook59 — APPROVED** [review]
:+1:

**Pull request merged**